presidio-analyzer/tests/test_analyzer_utils.py

```diff
@@ -1,6 +1,17 @@
 from presidio_analyzer import PresidioAnalyzerUtils
 import pytest
 
+
+@pytest.fixture(scope="module")
+def recognizer():
```

Comment:
Please rename this fixture to `presidio_analyzer_utils`. The name is confusing :)
Also, is this fixture used anywhere?